### DISCUSSION TITLE
8/4 안건

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#81] [01.Agenda]
# 안건
- 머지된 브랜치는 삭제해주세요🥺
<img width="385" alt="스크린샷 2022-08-03 오후 11 31 32" src="https://user-images.githubusercontent.com/72402747/182634670-039b76b2-318d-445f-82cf-eb2719ca59b9.png">

- 헤더 반응형 일부분을 고쳐야 할 것 같습니다!
![image](https://user-images.githubusercontent.com/74234333/182638832-d7407ec7-dabe-4151-9c9f-7ee34603dd4e.png)

-[맞춤전시회 라우팅](https://github.com/prgrms-web-devcourse/Team-BackFro-ArtZip-FE/discussions/82)

**Comment by punchdrunkard** [comment]
뜨끔해서 2개 지웠습니다.....
